Foundation/NSObjCRuntime.swift

```diff
@@ -206,6 +206,10 @@ internal func NSInvalidArgument(_ message: String, method: String = #function, f
     fatalError("\(method): \(message)", file: file, line: line)
 }
 
+internal func _NSMethodExceptionProem(_ obj: NSObject, _ cmd: StaticString = #function) -> String {
```

Comment:
this is taken from the actual Foundation; https://www.merriam-webster.com/dictionary/proem it is a preliminary comment